docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -24,32 +22,34 @@ NOTE: The CSPM feature currently only supports posture evaluations for Amazon We
 
 [discrete]
 [[cspm-add-and-name-integration]]
-=== Add your CSPM integration
+== Add the CSPM integration
 . From the Elastic Security *Get started* page, click *Add integrations*.
 . Search for `CSPM`, then click on the result.
 . Click *Add Cloud Security Posture Management (CSPM)*.
-. Give your integration a name that matches the purpose or team of the AWS account you want to monitor, for example, `dev-aws-account`.
+. Select *AWS*, then either *AWS Organization* to onboard multiple accounts, or *Single Account* to onboard an individual account.
+. Give your integration a name that matches the purpose or team of the AWS account/organization you want to monitor, for example, `dev-aws-account`.
+
 
 [discrete]
 [[cspm-set-up-cloud-access-section]]
-=== Set up cloud account access
+== Set up cloud account access
 The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access.
 
-For most users, the simplest option is to use AWS CloudFormation to automatically provision the necessary resources and permissions in your AWS account. This method, as well as several manual options, are described below.
+For most use cases, the simplest option is to use AWS CloudFormation to automatically provision the necessary resources and permissions in your AWS account. This method, as well as several manual options, are described below.
 
 [discrete]
 [[cspm-set-up-cloudformation]]
 === CloudFormation (recommended)
 . In the *Add Cloud Security Posture Management (CSPM) integration* menu, under *Setup Access*, select *CloudFormation*.
-. In a new browser tab or window, log in as an admin to the AWS account you want to onboard.
+. In a new browser tab or window, log in as an admin to the AWS account or organization you want to onboard.
 . Return to your {kib} tab. Click *Save and continue* at the bottom of the page.
 . Review the information, then click *Launch CloudFormation*.
-. A CloudFormation template appears in a new browser tab. You don't need to modify its configuration.
+. A CloudFormation template appears in a new browser tab.
+. For organization-level deployments only, you must enter the ID of the organizational unit where you want to deploy into the `OrganizationalUnitIds` field in the CloudFormation template. You can find it in the AWS console under *AWS Organizations -> AWS Accounts* (it appears under the organization name).
 . (Optional) Switch to the AWS region where you want to deploy using the controls in the upper right corner.
 . Tick the checkbox under *Capabilities* to authorize the creation of necessary resources.
 +
```

Comment:
The image below also needs updating since it doesn't reflect the current situation

1. The `Integration` field is not there anymore
2. For multi-account onboarding, `OrganizationalUnitIds` is also there

![screenshot-2023-09-13T09:41:00 CEST](https://github.com/elastic/security-docs/assets/5778622/94286b67-b037-4d86-adcc-fba929bae106)